homeassistant/components/jewish_calendar/__init__.py

```diff
@@ -39,49 +44,25 @@
     },
 }
 
-CONF_DIASPORA = "diaspora"
-CONF_LANGUAGE = "language"
-CONF_CANDLE_LIGHT_MINUTES = "candle_lighting_minutes_before_sunset"
-CONF_HAVDALAH_OFFSET_MINUTES = "havdalah_minutes_after_sunset"
-
-CANDLE_LIGHT_DEFAULT = 18
-
-DEFAULT_NAME = "Jewish Calendar"
-
-CONFIG_SCHEMA = vol.Schema(
-    {
-        DOMAIN: vol.Schema(
-            {
-                vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
-                vol.Optional(CONF_DIASPORA, default=False): cv.boolean,
-                vol.Inclusive(CONF_LATITUDE, "coordinates"): cv.latitude,
-                vol.Inclusive(CONF_LONGITUDE, "coordinates"): cv.longitude,
-                vol.Optional(CONF_LANGUAGE, default="english"): vol.In(
-                    ["hebrew", "english"]
-                ),
-                vol.Optional(
-                    CONF_CANDLE_LIGHT_MINUTES, default=CANDLE_LIGHT_DEFAULT
-                ): int,
-                # Default of 0 means use 8.5 degrees / 'three_stars' time.
-                vol.Optional(CONF_HAVDALAH_OFFSET_MINUTES, default=0): int,
-            }
-        )
-    },
-    extra=vol.ALLOW_EXTRA,
-)
+CONFIG_SCHEMA = vol.Schema({DOMAIN: vol.Schema(DATA_SCHEMA)}, extra=vol.ALLOW_EXTRA)
 
 
 async def async_setup(hass, config):
     """Set up the Jewish Calendar component."""
-    name = config[DOMAIN][CONF_NAME]
-    language = config[DOMAIN][CONF_LANGUAGE]
+    return True
```

Comment:
You need to store the configuration that is part of `config[DOMAIN]` if you want to keep supporting configuration.yaml. If you don't, update `CONFIG_SCHEMA` to only optionally accept an empty dict

Reply:
Copied from the abode integration adding support for both config_entries and configuration.yaml
Still failing tests. I suppose that fixing this should obsolete your comment on the tests?